### ISSUE TITLE
"MAILTO:" in uppercase in change_attendee_status

### ISSUE THREAD

**Issue by alexhook** [python-caldav/caldav#388]
Hi!

I have a problem during the attendee status change because some mail services have "MAILTO:" in uppercase. Can you please make the following change [here](https://github.com/python-caldav/caldav/blob/master/caldav/objects.py#L2431C9-L2431C71)?

from:
```python
strip_mailto = lambda x: str(x).replace("mailto:", "").lower()
```
to:
```python
strip_mailto = lambda x: str(x).lower().replace("mailto:", "")
```

**Comment by tobixen** [comment]
Could you create a pull request?

**Comment by tobixen** [comment]
I ran into this issue myself and fixed it in 42acfde4
